Content.Shared/CCVar/CCVars.cs

```diff
@@ -2294,6 +2294,13 @@ public static readonly CVarDef<float>
         public static readonly CVarDef<bool> HeightAdjustModifiesZoom =
             CVarDef.Create("heightadjust.modifies_zoom", false, CVar.SERVERONLY);
 
+        /// <summary>
+        ///     Whether height & width sliders adjust a player's bloodstream volume.
+        /// </summary>
+        /// <remarks>This can be configured more precisely by modifying BloodstreamAffectedByMassComponent.</remarks>
+        public static readonly CVarDef<bool> HeightAdjustModifiesBloodstream =
+            CVarDef.Create("heightadjust.modifies_bloodstream", true, CVar.SERVERONLY);
```

Comment:
```suggestion
        /// <summary>
        ///     Whether height & width sliders adjust a player's bloodstream volume.
        /// </summary>
        /// <remarks>
        ///     This can be configured more precisely by modifying BloodstreamAffectedByMassComponent.
        /// </remarks>
        public static readonly CVarDef<bool> HeightAdjustModifiesBloodstream =
            CVarDef.Create("heightadjust.modifies_bloodstream", true, CVar.SERVERONLY);
```